plugins/modules/rocketchat.py

```diff
@@ -100,6 +100,14 @@
     elements: dict
     description:
       - Define a list of attachments.
+  is_pre740:
+    description:
+      - If V(true), the payload matches Rocket Chat prior to 7.4.0 format.
```

Comment:
```suggestion
      - If V(true), the payload matches Rocket.Chat prior to 7.4.0 format. This format has been used by the module since its inception, but is no longer supported by Rocket.Chat 7.4.0.
```